endpoint/spec.md

```diff
@@ -318,38 +318,38 @@ See the [deprecated](../core/spec.md#deprecated) attribute in the core
 - Type: String
 - Description: The name of the specification that defines the Resource
   stored in the registry. Often it is difficult to unambiguously determine
-  what a Resource is via simple inspect of its serialization. This attribute
+  what a Resource is by simply inspecting its serialization. This attribute
```

Comment:
```suggestion
  what a Resource is by simply inspecting its serialized form. This attribute
```

(I view serialization as a process rather than a result that can be inspected.) Happy for this to be resolved with no action though.

Reply:
done